src/main/webapp/app/shared/model/skill-status.ts

```diff
@@ -0,0 +1,21 @@
+export const enum SkillStatus {
+    OPEN = 'OPEN',
+    ACHIEVED = 'ACHIEVED',
+    EXPIRING = 'EXPIRING',
+    EXPIRED = 'EXPIRED',
+    IRRELEVANT = 'IRRELEVANT'
+}
+
+export class SkillStatusUtils {
```

Comment:
I don't like the usage of a Util class for such methods. Just as a reference for future implementations:

```typescript
namespace SkillStatus {
    export isValid(skillStatus: SkillStatus): boolean {
        return skillStatus === SkillStatus.ACHIEVED || skillStatus === SkillStatus.EXPIRING;
    }
}
```
Then you could use the method like this:
```typescript
SkillStatus.isValid(variable)
```